src/features/vote/ActiveViewDetailsModal.tsx

```diff
@@ -38,24 +43,85 @@ interface Props {
   timestamp: string;
   ancData: string;
   unix: string;
+  committedVotes: VoteEvent[] | undefined | void;
+  revealedEvents: VoteRevealed[] | undefined | void;
```

Comment:
Why is there `void` here? I think its supposed to be used only expressions and is equal to `undefined` here

Reply:
You get a type error if you remove the void from useQuery. So it's one of the return types technically.

<img width="1093" alt="Screen Shot 2021-06-22 at 11 28 02 AM" src="https://user-images.githubusercontent.com/12792146/122979917-f2d5b880-d34c-11eb-814b-05709676a705.png">

Reply:
I see, I think that's because there is a `catch` in the queryFn, there is an `onError` callback that you can pass react-query to call `addError`, no need to change it though, I was just curious about the `void`